www/docs/mp/index.php

```diff
@@ -1162,11 +1156,11 @@ function policy_image($data, $MEMBER, $format) {
 }
 
 // generate party policy diffs
-function person_party_policy_diffs($MEMBER, $policiesList, $only_diffs) {
+function person_party_policy_diffs($MEMBER, $policiesList) {
     global $data;
 
     $divisions = new MySociety\TheyWorkForYou\Divisions($MEMBER);
-    $policySummaries = $divisions->getMemberDivisionDetails();
+    $policySummaries = $divisions->getMemberDivisionDetails($strong_only = true);
```

Comment:
Sorry, didn't notice this before - this should just be `(true)` (and the other same line), you don't want to set a variable here while calling the function.

Reply:
Doh - fixed in fixup.